awa-python/tests/test_dlq.py

```
@@ -0,0 +1,351 @@
+            run_lease,
+            max_attempts,
+            lane_seq,
+            run_at,
+            COALESCE(attempted_at, now()),
+            now(),
+            created_at,
+            unique_key,
+            unique_states,
+            payload
+        FROM moved
+        """,
```

Ruff (0.15.10)

[error] Possible SQL injection vector through string-based query construction

(S608)